### PR TITLE
Added new js Question

### PR DESCRIPTION
**Pull request by imashu001**

### CODE REVIEW

File: javascript/answers/imashu001-q35.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                    <h4>Explain window.onload and onDocumentReady?
+                    </h4>
+                </h4>
+            </div>
+            <br>
+            <br>
+            <div>
+                <strong>Answer:</strong> 
+                <p>
+                    The onload function is not run until all the information on the page is loaded. This leads to a substantial delay before any code is executed.
```
Comment (shivamyth) [inline javascript/answers/imashu001-q35.html:55]: Explain with an example

**Review by shivamyth — CHANGES_REQUESTED** [review]

**Comment by dakshkhetan** [comment]
@imashu001 Any updates on this? This PR will be closed in 24 hours if no changes are done.

**Review by imashu001 — COMMENTED** [review]
I have added code  Examples please have a look again

File: javascript/answers/imashu001-q35.html (hunk 1 above)
Comment (imashu001) [reply javascript/answers/imashu001-q35.html:55]: @dakshkhetan @shivamyth  I have added code Examples please have a look again

**Review by shivamyth — DISMISSED** [review]

**Review by shivamyth — APPROVED** [review]

**Pull request merged**